javascript/ql/src/experimental/SockJS/SockJS.qll

```diff
@@ -0,0 +1,47 @@
+/**
+ * Provides classes for working with [SockJS](http://sockjs.org).
+ */
+
+import javascript
+import DataFlow::PathGraph
+
+/**
+ * A model of the `SockJS` websocket data handler (https://sockjs.org).
+ */
+module SockJS {
+  class SourceFromSocketJS extends RemoteFlowSource {
+    /**
+     * Access to user-controlled data object received from websocket
+     * For example:
+     * ```
+     * server.on('connection', function(conn) {
+     *     conn.on('data', function(message) {
+     *       ...
+     *     });
+     *  });
+     * ```
+     */
+    SourceFromSocketJS() {
```

Comment:
```suggestion
  /**
   * Access to user-controlled data object received from websocket
   * For example:
   * ```
   * server.on('connection', function(conn) {
   *     conn.on('data', function(message) {
   *       ...
   *     });
   *  });
   * ```
   */
  class SourceFromSocketJS extends RemoteFlowSource {
    SourceFromSocketJS() {
```

The QLDoc should be above the class, not above the constructor inside the class.